examples/react-widget/subscribe.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
Sorry but it's broken. If I don't have any consents defined at all in the projects then widget is just stuck at `Loading consents...`. Please fix it.